schema/cam/cam_schema_2-1-0.json

```diff
@@ -0,0 +1,958 @@
+                      "default": 4095,
+                      "minimum": 0,
+                      "maximum": 4095
+                    },
+                    "semi_major_orientation": {
+                      "$comment": "if not provided, 'semi_major_orientation' = 3601 (unavailable)",
+                      "type": "integer",
+                      "description": "wgs84North(0), wgs84East(900), wgs84South(1800), wgs84West(2700), doNotUse(3600), unavailable(3601)",
+                      "default": 3601,
+                      "minimum": 0,
+                      "maximum": 3601
+                    }
+                  }
+                },
+                "altitude": {
+                  "type": "object",
+                  "required": [
+                    "altitude_value",
+                    "altitude_confidence"
+                  ],
+                  "properties": {
+                    "altitude_value": {
+                      "type": "integer",
+                      "description": "Unit: 0.01 meter. referenceEllipsoidSurface(0), oneCentimeter(1), unavailable(800001)",
+                      "default": 800001,
+                      "minimum": -100000,
+                      "maximum": 800001
+                    },
+                    "altitude_confidence": {
+                      "$comment": "if not provided, 'altitude_confidence' = 15 (unavailable)",
+                      "type": "integer",
+                      "description": "alt-000-01 (0), alt-000-02 (1), alt-000-05 (2), alt-000-10 (3), alt-000-20 (4), alt-000-50 (5), alt-001-00 (6), alt-002-00 (7), alt-005-00 (8), alt-010-00 (9), alt-020-00 (10), alt-050-00 (11), alt-100-00 (12), alt-200-00 (13), outOfRange (14), unavailable (15)",
+                      "default": 15,
+                      "minimum": 0,
+                      "maximum": 15
+                    }
+                  }
+                }
+              }
+            }
+          }
+        },
+        "high_frequency_container": {
+          "type": "object",
+          "oneOf": [
+            {
+              "required": [
+                "basic_vehicle_container_high_frequency"
+              ]
+            },
+            {
+              "required": [
+                "rsu_container_high_frequency"
+              ]
+            }
+          ],
+          "properties": {
+            "basic_vehicle_container_high_frequency": {
+              "type": "object",
+              "description": "The basic vehicle container high frequency",
+              "required": [
+                "heading",
+                "speed",
+                "drive_direction",
+                "vehicle_length",
+                "vehicle_width",
+                "longitudinal_acceleration",
+                "curvature",
+                "curvature_calculation_mode",
+                "yaw_rate"
+              ],
+              "properties": {
+                "heading": {
```

Comment:
If we transform each field into objects containing value and confidence (vs having a separate confidence object like before, which was a design choice), I would suggest that each object contains a short `value` and `confidence` field, rather than repeating the object name such as `heading_value` or `heading_confidence`.

Reply:
The new layout better matches the layout in the ETSI ASN spec, and so do the names of the properties.

I agree that this new layout is not, err..., very nice and convenient to work with, but I believe it would be diverging too much if we stayed with the previous layout. If ever, that should be done in a major version update where we actually completely depart from the ETSI spec). But for this minor update, I think we should stick to the layout and properties names as defined in the ETSI spec...

Reply:
I agree with following more closely the ETSI message layout, but I still think that simpler `value` / `confidence` pairs for each object would make it clearer and less clutered. CAMs being sent at high rates, it would still be nice to optimize messages a tiny bit (seeing `vehicle_length_value` and `vehicle_length_confidence_indication` scares me).

Reply:
OK, that makes sense. Let's go for shorter names where they make sense.